packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1076,4 +1076,14 @@ contract Validators is
     emit ValidatorDeaffiliated(validatorAccount, affiliation);
     return true;
   }
+
+  /**
+   * @notice Removes a validator from the group for which it is a member.
+   * @param validatorAccount The validator to remove and deaffiliate from their affiliated validator group.
+   */
+  function removeSlashedMember(address validatorAccount) external {
```

Comment:
Please add a modifier restricting who can call this

Reply:
If I understand the flow here, I'm assuming we want to create a `onlySlasher` modifier or something similar so that these fns can only be called from the Slasher contract, is that correct?